releasenotes/notes/add-c0f1d24cf11f23b7.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    - Botocore: Adds support for overriding the default service name in Botocore
```

Comment:
nit:
```suggestion
    - botocore: Adds support for overriding the default service name in `botocore`
```

Reply:
Oh I just saw that this is the exact opposite of what Zach suggested 😂 I leave it to you, it's a nit comment anyways!

Reply:
My bad I clearly read the opposite of what he meant, thanks for the catch!